src/slowmokit/methods/preprocessing/normalization.cpp

```diff
@@ -0,0 +1,32 @@
+/**
+ * @file methods/preprocessing/normalization.CPP
+ *
+ * Implementation of the Accuracy score main program
+ */
+#include "normalization.hpp"
+template <class T>
+void normalize(std::vector<T> &values)
+{
+    double minimum = *min_element(values.begin(), values.end());
```

Comment:
@YashitaBansal Replace double with T here as the datatype of min/max value will be the same as that in values vector